schwifty/bic.py

```diff
@@ -184,8 +184,14 @@ def from_bank_code(cls, country_code: str, bank_code: str) -> BIC:
             * Switzerland
         """
         try:
-            entries = cls.candidates_from_bank_code(country_code, bank_code)
-            return entries[0]
+            candidates = cls.candidates_from_bank_code(country_code, bank_code)
+            if len(candidates) > 1:
+                # If we have multiple candidates, we try to pick the
+                # one with XXX as branch code which is the most generic one.
+                generic_codes = [c for c in candidates if c.endswith("XXX")]
```

Comment:
What about 8 character BICs that don't even specify a branch code? Should we extend this to 
```suggestion
                generic_codes = [c for c in candidates if c.branch_code == "XXX" or not c.branch_code]
```

Reply:
Sure I like that, do you have one in mind for me to add a test for this case?